src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,51 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  title: string;
+  description: string;
+  imdbUrl: string;
+  imgUrl: string;
+};
+
+export const MovieCard: React.FC<Props> = (props) => {
+  const {
+    title,
+    description,
+    imdbUrl,
+    imgUrl,
+  } = props;
```

Comment:
як правило деструкторизація робиться одразу в пропсах?
```suggestion
export const MovieCard: React.FC<Props> = ({
    title,
    description,
    imdbUrl,
    imgUrl,
}) => {
```